features/admin.actions.v1/components/action-config-form.tsx

```diff
@@ -137,12 +132,13 @@ const ActionConfigForm: FunctionComponent<ActionConfigFormInterface> = ({
     );
 
     const renderLoadingPlaceholders = (): ReactElement => (
-        <>
-            <Skeleton variant="rectangular" className="placeholder label" />
-            <Skeleton variant="rectangular" className="placeholder text-field with-max-width" />
-            <Skeleton variant="rectangular" className="placeholder label" />
-            <Skeleton variant="rectangular" className="placeholder text-field with-max-width" />
-        </>
+
```

Comment:
remove this line

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/6774